Libs/Mesh/Mesh.h

```diff
@@ -36,6 +36,21 @@ class Mesh
   bool compare_points_equal(const Mesh& other_mesh);
   bool compare_scalars_equal(const Mesh& other_mesh);
 
+  // query functions //
+
+  /// number of vertices
+  vtkIdType numVertices() const { return mesh->GetNumberOfVerts(); }
+
+  /// number of faces
+  vtkIdType numFaces() const { return mesh->GetNumberOfCells(); }
+
+  /// get bounds
+  double* bounds() const {
+    double bb[6];
+    mesh->GetBounds(bb);
+    return bb;
```

Comment:
I see this warning in the build:

warning: address of stack memory associated with local variable 'bb' returned [-Wreturn-stack-address]
    return bb;

This array, 'bb' is allocated on the stack and will be gone as soon as the function returns.

Reply:
I have fixed this in the mesh_party branch. I created a Mesh::Region so that this does not happen.